regression-test/suites/sql_block_rule_p0/test_sql_block_rule.groovy

```diff
@@ -272,4 +272,17 @@ suite("test_sql_block_rule", "nonConcurrent") {
         """
     }
 
+    multi_sql """
+        drop SQL_BLOCK_RULE if exists rule_drop_r;
+
+        CREATE SQL_BLOCK_RULE rule_drop
+        PROPERTIES(
+        "sql"="select \\* from order_analysis",
```

Comment:
not get the purpose of this case

Reply:
without the change, sqlPattern will be 'NULL', so 'select NULL' would hit this sql block rule